### ISSUE TITLE
Normal Training + Sparsity training confusion

### ISSUE THREAD

**Issue by nscotto** [PengyiZhang/SlimYOLOv3#51]
Hello,

First of all, thank you for sharing the implementation. 
The **Normal training** section in the *REAME* is confusing me a bit about what is sparsity training, is sparsity training something to run post-normal training to get some numbers for the pruning step, or is it a modified training that needs to be run standalone?
On a practical level, the former implies possibility of using darknet for the normal training phase, while the latter implies full (sparsity) training in Pytorch.

The beautiful graph did not help my confusion.

Thank you.

**Comment by ykhsiao25** [comment]
Hi, I have the some question about this.
@PengyiZhang 
If normal training use "darknet53.conv.74.weights", should I use "darknet53.conv.74.weights" for sparsity training again?
Or use my own weights(ex:250best.pt, which is fine-tuned by darknet53.conv.74.weights)? 
But if I use my own weights, there is a "loss is nan" error
Thank you very much in advance,too.

**Comment by fatalfeel** [comment]
easy to told this
check https://arxiv.org/pdf/1907.11093.pdf in figure2
and
mapping to figure3
Procedure of learning efficient deep objectors for SlimYOLOv3

Pre-trained network in the first box = yolov3 
Pre-trained network is darknet53.conv.74.weights for Sparsity training
